src/cli/opts.js

```diff
@@ -62,12 +62,16 @@ export default async function parse(pkg, argv) {
   opts.include['copy-files'] = ['**/*']
   opts.exclude['copy-files'] = [...opts.include.js, ...opts.include.css, ...opts.exclude['*']]
 
+  if (process.env.NODE_ENV === 'production') {
```

Comment:
In that case, `NODE_ENV=production` wins. I did think about this and wrote it like so to be consistent. If you do `ez-build --interactive --production` (order is irrelevant) it does the same thing – turns off interactive in favor of production.

Thinking about it, this can get really confusing if you live in an environment where someone did `export NODE_ENV=production`, as you'll essentially never be able to turn production mode off when invoking ez-build:

```bash
$ export NODE_ENV=production
$ ez-build --interactive # Will still be set to production
```

This is no good. I'll change it, thanks buddy!